<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWidget</class>
 <widget class="QWidget" name="SettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="locale">
   <locale language="C" country="AnyCountry"/>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="labelLanguage">
     <property name="text">
      <string>Language</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>comboLanguage</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboLanguage"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelGlWidget">
     <property name="text">
      <string>Type of map</string>
     </property>
     <property name="buddy">
      <cstring>comboGlWidget</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboGlWidget">
     <item>
      <property name="text">
       <string>2d - Cylindrical projection</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3d - Earth</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelStyle">
     <property name="text">
      <string>Style</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>comboStyle</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="comboStyle"/>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="checkPalette">
     <property name="text">
      <string>Use standart palette</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="checkUpdates">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Check for updates</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="checkSave">
     <property name="text">
      <string>Save settings</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QPushButton" name="btnPrintScr">
     <property name="text">
      <string>Snapshot</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>379</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QPushButton" name="btnReset">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Reset to default</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboLanguage</tabstop>
  <tabstop>comboGlWidget</tabstop>
  <tabstop>comboStyle</tabstop>
  <tabstop>checkPalette</tabstop>
  <tabstop>checkUpdates</tabstop>
  <tabstop>checkSave</tabstop>
  <tabstop>btnReset</tabstop>
  <tabstop>btnPrintScr</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
